src/main/java/net/starlark/java/eval/StringModule.java

```diff
@@ -191,6 +191,11 @@ public String upper(String self) {
           "\u0009" + "\n" + "\u000B" + "\u000C" + "\r" + "\u001C" + "\u001D" + "\u001E" + "\u001F "
               + "\u0085" + "\u00A0");
 
+  private static final CharMatcher ASCII_WHITESPACE =
```

Comment:
Add javadoc indicating that we use this when using the latin-1 hack in order to avoid triggering on utf-8 code units above 0x7f.